docs/VLSI/Tutorial.rst

```diff
@@ -0,0 +1,141 @@
+.. _tutorial:
+
+ASAP7 Tutorial
+==============
+The ``vlsi`` folder of this repository contains an example HAMMER flow with the SHA-3 accelerator and a dummy hard macro in the ASAP7 PDK. It is intended for use with the Cadence and Mentor tool plugins.
```

Comment:
Maybe it would be good to use explicit hammer terminology here:
"This example tutorial uses an ASAP7 Technology plug-in and a Cadence CAD plug-in."
Where does the Mentor tool plug-in fit here?

Reply:
Sure. The Mentor tool plug-in is needed for DRC & LVS.

Reply:
So let's say that. Because I think it's reasonable to assume that some people may have Cadence tools but won't have Mentor tools, and they should know how far in the flow they can get